contribs/gmf/less/desktop.less

```diff
@@ -20,6 +20,10 @@
 @toolbar-width: 4.5rem;
 @border-color: darken(@brand-primary, @standard-variation);
 @search-width: 8 * @map-tools-size;
+@font-size-base: 13px;
+@padding-base-vertical: 5px;
+@padding-base-horizontal: 10px;
+@form-group-margin-bottom: 10px;
```

Comment:
Bootstrap stable release is not intended to be released sometimes soon. Let's keep pixels for now to match Bootstrap 3 for now. It should be easy to change this when we migrate to 4.